packages/core/src/storage/migration/__tests__/backup.test.ts

```diff
@@ -130,7 +130,8 @@ describe('UpdateAssistant | Backup', () => {
 
     try {
       await updateAssistant.update()
-    } catch (error) {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    } catch (error: any) {
```

Comment:
Any chance we can avoid this? unknown or just no type?

Reply:
Just no type I guess. There's also the option to add "useUnknownInCatchVariables": true to tsconfig (But that gets a little weird sldo depending on typescript version aka works for me locally but fails in the gh pipeline. I'll just revert it to previous for now.